rocky-raft/src/rocky/raft/dto/ServerConfig.java

```diff
@@ -0,0 +1,18 @@
+package rocky.raft.dto;
+
+public class ServerConfig extends Address {
```

Comment:
Logically ServerConfig is not a "parent" of Address. ServerConfig should contain "id" and "address". It'll be easier if we let the id be integer (because we assume it at a lot of places). Just need to make sure anywhere we use Config.SERVERS.get(index), we use Config.get(id) instead.